app/config/config.php

```diff
@@ -9,7 +9,7 @@
     define('APPROOT', dirname(dirname(__FILE__)));
 
     //URLROOT (Dynamic links)
-    define('URLROOT', 'http://www.magazijnmboutrecht.nl');
+    define('URLROOT', 'magazijnproject.org');
```

Comment:
Why is this changed?

Reply:
Had an issue that I couldn't get around, Jesse fixed it by doing this and recommended we implement it project wide to avoid more issues than neecesary